src/components/StatsPanel/StatsPanel.constants.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  WorkOutline,
+  CheckCircleOutline,
+  ThumbDownOffAlt,
+  HourglassEmpty,
+} from '@mui/icons-material';
+
+import type { StatItemData } from './StatsPanel.types';
+
+export const TEST_STATS: Record<string, StatItemData> = {
```

Comment:
TEST_STATS is typed as Record<string, StatItemData>, which is broad and loses static guarantees about the expected keys (total, passed, failed, waiting). This allows typos in keys to go unnoticed and reduces type-safety when indexing TEST_STATS in the hook.

**Sugerencia:**
```typescript
Narrow the type so the keys are explicit. For example:

export const TEST_STATS: Record<'total' | 'passed' | 'failed' | 'waiting', StatItemData> = {
  ...
};

Or let TypeScript infer the shape by removing the explicit Record type and using a const assertion if appropriate.
```